### PR TITLE
Added Geo location App

### PR DESCRIPTION
**Pull request by Sloth-Panda**
## Related Issue
  - Info about Issue or bug
issue : #108 

### Describe the changes you've made
A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!
Made a geo location app the shows ur position on  a google map using maps api and it also has proper error handling

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.
NA
### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
Put any screenshot(s) of the project here.
Screenshot (145)
Screenshot (146)

### CODE REVIEW

**Comment by Sloth-Panda** [comment]
@urvashi-code1255  @Vishal-raj-1

**Comment by Vishal-raj-1** [comment]
@Sloth-Panda I can't merge any PR which consists of conflicts file. Let  me explain how you can solve this issue 

first of all do `git pull upstream master`  then do your changes and then do `git push`.

**Comment by Sloth-Panda** [comment]
@Vishal-raj-1 is not solving the merge conflicts  from  here only and doing  `commit merge` wont work?

**Comment by Vishal-raj-1** [comment]
@Sloth-Panda Good I was waiting for your update !!

File: assets/css/location.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
```
Comment (Vishal-raj-1) [inline assets/css/location.css:1]: aah ha !! @Sloth-Panda Please format this file.

**Review by Vishal-raj-1 — COMMENTED** [review]
You haven't add this project on main website.

**Comment by urvashi-code1255** [comment]
Hey @Sloth-Panda Any updates on this?

**Comment by Sloth-Panda** [comment]
Okay I will make a pr soon

**Comment by urvashi-code1255** [comment]
Any updates @Sloth-Panda ?

**Comment by Sloth-Panda** [comment]
Hey, I got busy with something. will be  making prs by tomorrow

**Pull request closed without merging**